src/system.c

```diff
@@ -25,6 +25,16 @@ void emu_update_uart_interrupts(riscv_t *rv)
     plic_update_interrupts(attr->plic);
 }
 
+#define READ 1
+#define WRITE 0
```

Comment:
The naming scheme might be too generic to identify.

Reply:
> The naming scheme might be too generic to identify.

How about `READ_MODE` / `WRITE_MODE`?

Reply:
> How about `READ_MODE` / `WRITE_MODE`?

Instead, you can define two macros, `CLEAR_PENDING_SIGNAL_R` and `CLEAR_PENDING_SIGNAL_W`.